### ISSUE TITLE
iNat21 (groovy-grape)

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#10]
Pre-train swin-v2-base on iNat21 with hierarchical multitask for 90 epochs at 192x192, then 30 epochs at 256x256.

For the 90 epochs, use a lower learning rate (1.25e-4).

**Comment by samuelstevens** [comment]
[wandb groovy-grape-256](https://wandb.ai/samuelstevens/hierarchical-vision/runs/11o47wpm?workspace=user-samuelstevens)
[wandb haunted-broomstick-17](https://wandb.ai/samuelstevens/hierarchical-vision/runs/28ff9k82?workspace=user-samuelstevens) was the previous run before I adopted the fruit naming scheme.

**Comment by samuelstevens** [comment]
Finished pre-training. Available on AWS S3 at `s3://imageomics-models/groovy-grape-192-epoch89.pth`
